models/classes/TaoLtiSession.php

```diff
@@ -28,27 +26,33 @@
 use oat\taoLti\models\classes\ResourceLink\LinkService;
 use oat\taoLti\models\classes\user\LtiUserInterface;
 
-/**
- * The TAO layer ontop of the LtiSession
- *
- * @access public
- * @author Joel Bout, <joel@taotesting.com>
- * @package taoLti
- */
 class TaoLtiSession extends common_session_DefaultSession
 {
-    /**
-     * @var core_kernel_classes_Resource
-     */
-    private $ltiLink = null;
+    private const VERSION_LTI_1P1 = '1.1';
+    private const VERSION_LTI_1P3 = '1.3';
+
+    /** @var string */
+    protected $linkId = null;
```

Comment:
Any reason to make keep it protected instead of private?  Same question for https://github.com/oat-sa/extension-tao-lti/pull/313/files#diff-53587515f34586f56ff6e3b63ffc0e2131ccf8dacbb95a9dd2c7a69ac0ed9290R38

Reply:
No reason, it's committed by mistake. Thank you for spotting that. :)